### PR TITLE
Add io.github.cxong.cdogs-sdl

### PR DESCRIPTION
**Pull request by cxong**

### CODE REVIEW

File: io.github.cxong.cdogs-sdl.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "app-id": "io.github.cxong.cdogs-sdl",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
```
Comment (LeandroStanger) [inline io.github.cxong.cdogs-sdl.json]: Update runtime to 19.08

**Comment by LeandroStanger** [comment]
bot, build io.github.cxong.cdogs-sdl

File: io.github.cxong.cdogs-sdl.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "sdk": "org.freedesktop.Sdk",
+  "command": "cdogs-sdl",
+  "rename-icon": "cdogs-sdl",
+  "rename-appdata": "cdogs-sdl",
+  "rename-desktop-file": "cdogs-sdl.desktop",
+  "rename-appdata-file": "cdogs-sdl.appdata.xml",
```
Comment (LeandroStanger) [inline io.github.cxong.cdogs-sdl.json]: New appdata files should use metainfo instead. freedesktop.org/software/appstream/docs/chap-Quickstart.html

File: cdogs-sdl.appdata.xml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop">
```
Comment (bilelmoussaoui) [inline cdogs-sdl.appdata.xml]:
```suggestion
<component type="desktop-application">
```

File: cdogs-sdl.appdata.xml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  <id>cdogs-sdl.desktop</id>
```
Comment (bilelmoussaoui) [inline cdogs-sdl.appdata.xml]:
```suggestion
  <id>io.github.cxong.cdogs-sdl</id>
```

Comment (bilelmoussaoui) [reply cdogs-sdl.appdata.xml]: You need a `<launchable type="desktop-id">io.github.cxong.cdogs-sdl.desktop</launchable>` too.

File: cdogs-sdl.desktop [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+GenericName=Action/Arcade Game
+Version=1.0
+Type=Application
+Categories=Game;ActionGame;
+Keywords=Game;Arcade;Shooter;Retro;
+Comment=Classic overhead run-and-gun game
+TryExec=cdogs-sdl
+Exec=cdogs-sdl
+Terminal=false
+Icon=cdogs-sdl
```
Comment (bilelmoussaoui) [inline cdogs-sdl.desktop]:
```suggestion
Icon=io.github.cxong.cdogs-sdl
```
Also rename the file & remove the rename-* things from the manifest.

File: io.github.cxong.cdogs-sdl.json (hunk 2 above)
Comment (bilelmoussaoui) [inline io.github.cxong.cdogs-sdl.json]: You can just install them with the correct names.

**Comment by bilelmoussaoui** [comment]
The appdata and desktop files should be upstream

**Comment by cxong** [comment]
Thanks for the review @LeandroStanger and @bilelmoussaoui ! Some of the fixes will require upstream changes so I'll hold off this PR for now and recreate it after the next upstream release.

**Comment by LeandroStanger** [comment]
bot, build io.github.cxong.cdogs-sdl

**Pull request closed without merging**